examples/port-2-alibaba2-connection/terraform.tfvars.example

```diff
@@ -0,0 +1,35 @@
+equinix_client_id      = "MyEquinixClientId"
+equinix_client_secret  = "MyEquinixSecret"
+
+connection_name             = "Port_2_alibaba"
+connection_type             = "EVPL_VC"
+notifications_type          = "ALL"
+notifications_emails        = ["example@equinix.com"]
+bandwidth                   = 50
+purchase_order_number       = "1-323292"
+aside_port_name             = "sit-tb1-dc-e5.tlab,10GSMF,A,001,201257, 21951980"
+aside_vlan_tag              = 2019
+zside_ap_type               = "SP"
+zside_ap_authentication_key = "<Alibaba Auth Key>"
+zside_ap_profile_type       = "L2_PROFILE"
+zside_location              = "SY"
+zside_sp_name               = "Alibaba Cloud Express Connect"
+zside_seller_region         = "ap-southeast-2"
+zside_ap_profile_type       = "L2_PROFILE"
+zside_ap_type               = "SP"
+zside_location              = "SV"
+zside_seller_region         = "us-west-1"
+zside_sp_name               = "Alibaba Cloud Express Connect"
+region                      = "us-west-1"
+local_gateway_ip            = "10.0.0.1"
+peer_gateway_ip             = "10.0.0.2"
+peering_subnet_mask         = "255.255.255.252"
+virtual_border_router_name  = "Port_2_Alibaba_VBR"
+min_rx_interval             = 1000
+min_tx_interval             = 1000
+detect_multiplier           = 10
+access_key                  = "<Alibaba_Cloud_Access_Key>"
+secret_key                  = "<Alibaba_Cloud_Secret_Key>"
+project_id                  = "<Fabric_Project_ID>"
+physical_connection_id      = "<Physical_Connection_ID>"
+vlan_id                     = 2019
```

Comment:
This is computed inside of the main.tf because it has to match what equinix created. Should be removed.